dart/lib/src/protocol/sentry_feature_flag.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:meta/meta.dart';
+
+import 'access_aware_map.dart';
+
+@immutable
```

Comment:
are we keeping this immutable even after the mutable data PR merge?

Reply:
No, just wanted to stay compliant with the current main, in case we decide to postpone the othere PR or something similar.

Reply:
gotcha, I think we will merge it pretty soon